### PR TITLE
[openui5] Update the definition files for OpenUI5 1.118

### PR DESCRIPTION
**Pull request by akudev**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).  → as detailed in the [previous pull request for 1.90](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53680), some are unavoidable at this point
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: → no specific URL about the changes, it's just a new release: https://sdk.openui5.org/1.118.0
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by petermuessig — APPROVED** [review]

File: types/openui5/sap.ui.core.d.ts [hunk 1]
```diff
@@ -11066,32 +11226,9 @@ declare module "sap/ui/core/library" {
   }
 
   /**
-   * @since 1.10
-   *
-   * Defines the different message types.
+   * Specifies possible message types.
    */
-  export enum MessageType {
```
Comment (codeworrior) [inline types/openui5/sap.ui.core.d.ts:11073]: incompatible loss of APIs, caused by SAP/openui5@d659543

File: types/openui5/sap.ui.core.d.ts [hunk 2]
```diff
@@ -28480,101 +28995,6 @@ declare module "sap/ui/core/message/MessageManager" {
      * @returns Metadata object describing this class
      */
     static getMetadata(): Metadata;
-    /**
-     * Add messages to MessageManager
-     */
-    addMessages(
```
Comment (codeworrior) [inline types/openui5/sap.ui.core.d.ts:28486]: incompatible loss of APIs, caused by SAP/openui5@d65954372

File: types/openui5/sap.ui.webc.common.d.ts [hunk 3]
```diff
@@ -81,24 +140,6 @@ declare module "sap/ui/webc/common/WebComponentMetadata" {
        */
       oClassInfo: object
     );
-
```
Comment (codeworrior) [inline types/openui5/sap.ui.webc.common.d.ts:84]: Also looks incompatible, reason unclear...

**Review by codeworrior — COMMENTED** [review]
Comment on issues reported via internal incident 2370090386.

**Pull request merged**